druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -755,28 +779,16 @@ private void checkGroupBySingleSortLimit(boolean approx) {
   /** Tests a query that contains no GROUP BY and is therefore executed as a
    * Druid "select" query. */
   @Test public void testFilterSortDesc() {
-    final String sql = "select * from \"foodmart\"\n"
```

Comment:
I added another test case to test for this, instead of reverting the existing ones because I need to use wiki to test it.